metaworld/policies/sawyer_sweep_into_goal_v1_policy.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+
+from metaworld.policies.action import Action
+from metaworld.policies.policy import Policy, assert_fully_parsed, move
+
+
+class SawyerSweepIntoGoalV1Policy(Policy):
```

Comment:
is this the canonical name? isn't it just `sweep-into`?

Reply:
`env_dict` calls it `sweep-into-v1`, but the file name of the environment is `sawyer_sweep_into_goal.py`

Reply:
please stick to the names in env_dict